LL/section_135.trig

```diff
@@ -0,0 +1,397 @@
+@base <http://example.com/LL/section_135.trig>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix foaf:  <http://xmlns.com/foaf/0.1/> .
+@prefix rel: <http://purl.org/vocab/relationship/>.
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix dctype: <http://purl.org/dc/dcmitype/> .
+@prefix irishRel: <http://example.com/earlyIrishRelationship.ttl#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+
+<http://example.com/LL> {
```

Comment:
Do we know anything about the scribal hand or textual history of this tract? And does it appear in O'Brien's edition? Perhaps we should group it in a separate sub-folder from the LL text of CGH, if there is a possibility that it is a distinct text.

Reply:
Having a look, this is not in CGH, as CGH ends on f341 of LL.  Also, this is not CGH, this is the typed up version of Bergin and Best's diplomatic edition of LL (https://celt.ucc.ie//published/G800011F/header.html) so I would hesitate to equate it to CGH.

Reply:
Whether or not O'Brien included in CGH might indicate whether it is normally part of that collection (e.g. maybe this section is part of CGH proper in the Book of Lecan or suchlike) or whether it is an independent tract. This is not a very relevant point, I suppose, as CGH's material is not itself internally harmonious! But we need to decide, for the purpose of queries, is this tract to have its testimony presented as part of that of LL's CGH or as a separate text (just like we have not made Accallam na Senórach part of the Laud genealogies proper, in terms of file structure).

Reply:
Does Schlueter mention it, by any chance?

Reply:
I think we should move this to email as I have closed this PR. I think it may get confusing to resurrect this PR.